lib/hawkular/base_client.rb

```diff
@@ -42,7 +42,7 @@ def initialize(entrypoint = nil,
     end
 
     def url(url_format, *params)
-      url_format % params.map { |p| ERB::Util.url_encode(p) }
+      url_format % params.map { |p| hawk_escape(p) }
```

Comment:
Did this fail for some cases?

Reply:
oops, I didn't mean to commit that... good catch!